Vagrantfile

```diff
@@ -360,11 +351,12 @@ Vagrant.configure(2) do |config|
   end
 
   config.vm.define "darwin64" do |b|
-    b.vm.box = "jhcook/macos-sierra"
+    b.vm.box = "yzgyyang/macOS-10.14"
     b.vm.provider :virtualbox do |v|
       v.memory = 2048 + $wmem
-      v.customize ['modifyvm', :id, '--ostype', 'MacOS1012_64']
+      v.customize ['modifyvm', :id, '--ostype', 'MacOS_64']
       v.customize ['modifyvm', :id, '--paravirtprovider', 'default']
+      v.customize ['modifyvm', :id, '--nested-hw-virt', 'on']
```

Comment:
is this needed? for what?

does that make problems on older cpus?

Reply:
`MacOS_64` worked for me for all future versions as well. The version-specific OS settings in Virtualbox seem to stop after 10.13. Since `MacOS_64` works with old *and* new versions, I'd say it's more future-proof and no settings need to be changed if the VM is updated at some point.

Reply:
ok